src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -228,17 +230,14 @@ export const PublishingAccordion = ({
                     : ''
                 : t('registration.public_page.tasks_panel.registration_will_soon_be_published')}
             </Typography>
-            <LoadingButton
-              variant="contained"
-              color="info"
-              size="small"
-              loading={isLoadingData}
-              fullWidth
-              onClick={refetchData}
-              startIcon={<RefreshIcon />}
-              data-testid={dataTestId.registrationLandingPage.tasksPanel.refreshPublishingRequestButton}>
-              {t('registration.public_page.tasks_panel.reload')}
-            </LoadingButton>
+            <RefreshPublishingRequestButton refetchData={refetchData} isLoadingData={isLoadingData} />
+          </>
+        )}
+
+        {isDeletedRegistration && isWaitingForFileDeletion && (
```

Comment:
Statussjekk er en del av `isWaitingForFileDeletion` også

```suggestion
        {isWaitingForFileDeletion && (
```